docs/content/rules/sort-array-includes.mdx

```diff
@@ -192,6 +192,11 @@ Allows you to group array elements by their kind, determining whether spread val
 
 ### partitionByComment
 
+<sub>
+  - type:
+    - `boolean | string | string[]` (`BaseType`)
```

Comment:
Are you sure we need this block? We just have default values in other places.

Reply:
@hugop95 What do you think?

Reply:
@azat-io We can keep `boolean | string | string[]` but, we also need to say that users can also write `{ block: string | string[] | boolean; line: string | string[] | boolean }`.

We can remove that block and go with

```ts
### partitionByComment

<sub>default: `false`</sub>

Allows you to use comments to separate the exports into logical groups. This can help in organizing and maintaining large export blocks by creating partitions based on comments.

- `true` — All comments will be treated as delimiters, creating partitions.
-	`false` — Comments will not be used as delimiters.
- `string` — A regexp pattern to specify which comments should act as delimiters.
- `string[]` — A list of regexp patterns to specify which comments should act as delimiters.
- `{ block: string | string[] | boolean; line: string | string[] | boolean }` — Specify which block and line comments should act as delimiters.
```

even though the last line is a bit long. What do you think?

Reply:
I think it makes more sense that way.

Reply:
@azat-io Updated in 566872ab88a737733a76b15ac7d6f6144216ba35!